packages/sync-server/src/sync/snapshotOutgoingChanges.ts

```diff
@@ -62,9 +59,9 @@ export const snapshotOutgoingChanges = async (
         AND (
           project_ids IS NULL
           OR
-          project_ids::text[] && ARRAY[${projectIds.map(p => `?`).join(',')}]
+          project_ids::text[] && ARRAY[${projectIds.map(_p => `?`).join(',')}]
```

Comment:
```suggestion
          project_ids::text[] && ARRAY[${projectIds.map(() => `?`).join(',')}]
```
Does this get linted against? Or you find it less clear?